### PR TITLE
Displayable string language/dir support from document context

### PR DESCRIPTION
**Pull request by ianbjacobs**
Borrow language from the Payment Request API Proposed Recommendation regarding inheritance of language and direction information from the document for use in the browser's native UX.

Intended to address #93

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/secure-payment-confirmation/pull/192.html" title="Last updated on Jun 23, 2022, 3:20 PM UTC (50941a3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/secure-payment-confirmation/192/12f96d5...50941a3.html" title="Last updated on Jun 23, 2022, 3:20 PM UTC (50941a3)">Diff</a>

### CODE REVIEW

File: spec.bs
Comment (stephenmcgruer) [inline spec.bs:695]:
```suggestion
NOTE: The language and direction of the {{CollectedClientAdditionalPaymentData/payeeName} and {{PaymentCredentialInstrument/displayName}} may be determined from information inherited from the document. For discussion about enhanced support in JavaScript for language and direction of these displayable strings, see<a href="https://github.com/w3c/secure-payment-confirmation/issues/93">issue 93</a>.
```

(Remove often).

**Review by stephenmcgruer — APPROVED** [review]
LGTM modulo one comment

**Comment by ianbjacobs** [comment]
Thanks, Stephen. I've folded in your suggestion. Let's wait to hear from I18N folks before merging.

**Comment by ianbjacobs** [comment]
Closing this based on 
 https://github.com/w3c/secure-payment-confirmation/issues/209

**Pull request closed without merging**
